src/CAPI.js

```diff
@@ -50,7 +50,8 @@ define(['authAgents/SessionAuthAgent', 'authAgents/HttpBasicAuthAgent', 'Connect
             connectionStack: [ // Array of connections, should be filled-in in preferred order
                 {connection: XmlHttpRequestConnection},
                 {connection: MicrosoftXmlHttpRequestConnection}
-            ]
+            ],
+            siteAccess: null
```

Comment:
this option deserves to be documented in the CAPI constructor. It's also a good opportunity to improve the documentation of options with something like:

```js
     * @param [options] {Object} Object containing different options for the CAPI
     * @param [options.rootPath='/api/ezp/v2/'] {String} the API root path
     * @param [options.logRequests=false] {Boolean} whether to log requests
     * @param [options.siteAccess=null] {String|null} siteaccess in which requests should be executed
```